aws-msk/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: template-quickstart
+
+# Displayed in the UI (required)
+title: Template Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  The template quickstart allows you to get visibility into the performance and available of your example service and dependencies. Use this quickstart together with the mock up integrations.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A short form description for this quickstart
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - John Smith
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - list
```

Comment:
You'll need to add some keywords here so that the quickstart shows up in the categories. May I suggest `aws` and `cloud`?